core/src/main/resources/openapi/web3signer-eth2.yaml

```diff
@@ -533,6 +533,8 @@ components:
       properties:
         version:
           type: string
+          description: 'version to identify block request type.'
```

Comment:
Perhaps define the version as an enum so all the possible values are defined somewhere?